presto-spi/src/test/java/com/facebook/presto/spi/resourceGroups/TestResourceGroupId.java

```diff
@@ -28,6 +30,14 @@ public void testBasic()
     }
 
     @Test
+    public void testCodec()
```

Comment:
Add a test that compares to a fixed JSON string (so we can see the exact representation and make sure it doesn't accidentally change).